packages/web-frontend/src/screens/LandingPage/LandingPage.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * This is just a place holder Landing Page template. The actual landing page template will be
+ * developed in another ticket. @see waitp-1189
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import MuiContainer from '@material-ui/core/Container';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Button from '@material-ui/core/Button';
+import OverlayDiv from '../../containers/OverlayDiv';
+import { useAuth } from './useAuth';
+import { useNavigation } from './useNavigation';
+import { ProjectCardList } from '../../containers/OverlayDiv/components/ProjectPage/ProjectCardList';
+import { useCustomLandingPages } from './useCustomLandingPages';
+
+const Wrapper = styled.div`
+  position: relative;
+  background-size: cover;
+  background-position: center;
+`;
+
+const Container = styled(MuiContainer)`
+  position: relative;
+  min-height: calc(100vh - 75px);
+  background-color: #bfbfbf;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
Very minor, but would it be possible use `rem`/`em`/`percentages` instead of `px` values, so that will help us down the line with responsive styling?